### ISSUE TITLE
False positives

### ISSUE THREAD

**Issue by feanorknd** [coreruleset/nextcloud-rule-exclusions-plugin#5]
Hi there, lots of 921110 errors...

``
ModSecurity: Warning. Matched "Operator `Rx' with parameter `(?:get|post|head|options|connect|put|delete|trace|track|patch|propfind|propatch|mkcol|copy|move|lock|unlock)\s+(?:\/|\w)[^\s]*(?:\s+http\/\d|[\r\n])' against variable `REQUEST_BODY' (Value: `<?xml version="1.0" ?>\x0a<d:propfind xmlns:d="DAV:">\x0a  <d:prop>\x0a    <d:getetag/>\x0a  </d:pro (23 characters omitted)' ) [file "/etc/modsecurity/owasp/coreruleset/rules/REQUEST-921-PROTOCOL-ATTACK.conf"] [line "33"] [id "921110"] [rev ""] [msg "HTTP Request Smuggling Attack"] [data "Matched Data: propfind xmlns:d="dav:">\x0a found within REQUEST_BODY: <?xml version="1.0" ?>\x0a<d:propfind xmlns:d="dav:">\x0a  <d:prop>\x0a    <d:getetag/>\x0a  </d:prop>\x0a</d:propfind>\x0a"] [severity "2"] [ver "OWASP_CRS/3.3.2"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag "attack-protocol"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "capec/1000/210/272/220/33"] [hostname "xxxxxxxxx"] [uri "/remote.php/dav/files/a.xxxxxxxxx/"] [unique_id "166756463217.934518"] [ref "o26,25v844,105t:urlDecodeUni,t:htmlEntityDecode,t:lowercase"]
``

just detecting:

```
---fIkF7ECV---C--
<?xml version="1.0" ?>
**<d:propfind xmlns:d="DAV:">**
  <d:prop>
    <d:getetag/>
  </d:prop>
</d:propfind>
```

Maybe could be good to include an exception for this.

Thanks.

**Comment by theseion** [comment]
Same issue as https://github.com/coreruleset/coreruleset/issues/2973, but for the new plugin.

**Comment by EsadCetiner** [comment]
@feanorknd I created a pull request https://github.com/coreruleset/nextcloud-rule-exclusions-plugin/pull/42 to fix FPs relating to file uploads, could you let me know if this fixes your issue?
